components/common/footer/Footer.tsx

```diff
@@ -1,6 +1,6 @@
 export default function Footer() {
   return (
-    <footer className="text-white body-font bg-gradient-to-r from-blue-500 via-purple-600 to-red-500">
+    <footer className="text-white body-font bg-gradient-to-r from-blue_fcsc via-orange to-blue_fcsc">
```

Comment:
@sithumonline can't we use `-` instead `_`?

Reply:
We cannot use `-` on json

Reply:
Ohh, Okay then